impeller/renderer/backend/vulkan/texture_source_vk.h

```diff
@@ -125,12 +125,40 @@ class TextureSourceVK {
   ///
   virtual bool IsSwapchainImage() const = 0;
 
+  // These methods should only be used by render_pass_vk.h
```

Comment:
It would have to be friend for the class and all of the test classes. TBH I haven't figured out how to do that consistently for test classes, since the name is macro generated?